===================================================================
RCS file: /home/cvs/opie/libopie2/opiecore/device/odevice_zaurus.cpp,v
retrieving revision 1.48
retrieving revision 1.49
diff -u -p -r1.48 -r1.49
--- libopie2/opiecore/device/odevice_zaurus.cpp	2005/09/03 18:01:49	1.48
+++ libopie2/opiecore/device/odevice_zaurus.cpp	2006/02/06 13:40:28	1.49
@@ -242,9 +242,6 @@ void Zaurus::init(const QString& cpu_inf
         case Model_Zaurus_SL5500:
             m_backlightdev = "/sys/class/backlight/locomo-backlight/";
             break;
-        case Model_Zaurus_SL6000:
-            m_backlightdev = "/sys/class/backlight/tosa-bl/";
-            break;
         default:
             m_backlightdev = "/sys/class/backlight/corgi-bl/";
     }
